src/components/markdown/Tabs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Tabs({ markdownFiles }) {
+  const markdownList = markdownFiles.map((markdown, i) => {
+    return <li key={i}>{markdown}</li>;
+  });
+
+  return (
+    <>
+    <ul>
+      <li>{markdownList}</li>
```

Comment:
```suggestion
      {markdownList}
```
Lines 5-7 take care of creating the li element.

Reply:
implemented comment